ldaptor/numberalloc.py

```diff
@@ -29,7 +30,7 @@ def _nextGuess(self, found, lastGuess):
         if max is None:
             max=self.min+1000
 
-        guess=(max+self.min)/2
+        guess = (max + self.min) // 2
```

Comment:
why double (//) instead of single (/) ?

Reply:
To keep the same behaviour between versions

```
Python 2.7.12 (default, Nov 20 2017, 18:23:56) 
>>> 10/3
3

Python 3.5.2 (default, Nov 23 2017, 16:37:01) 
>>> 10/3
3.3333333333333335
>>> 10//3
3
```

Reply:
O.o  really?  These kinds of things drives me nuts...